demo/src/Resource/Page/User.php

```
@@ -9,10 +9,8 @@
 
 class User extends ResourceObject
 {
-    /**
-     * @Embed(rel="user", src="/api/user{?id}")
-     */
-    public function onGet($id)
+    #[Embed(rel: 'user', src: '/api/user{?id}')]
+    public function onGet(int $id): static
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)